src/dom/Draggable.js

```diff
@@ -49,12 +49,14 @@ export var Draggable = Evented.extend({
 
 	// @constructor L.Draggable(el: HTMLElement, dragHandle?: HTMLElement, preventOutline?: Boolean, options?: Draggable options)
 	// Creates a `Draggable` object for moving `el` when you start dragging the `dragHandle` element (equals `el` itself by default).
-	initialize: function (element, dragStartTarget, preventOutline, options) {
+	initialize: function (element, dragStartTarget, preventOutline, options, map, anchor) {

```

Comment:
Too many arguments.
May be better put them into `options`.

Also I have concerns about some edge cases: what if draggable is detached from initial map and reattached to another in run time?

Reply:
There is a problem when draggable is reattached to another map with a different angle.